bower.json

```diff
@@ -8,9 +12,9 @@
   "devDependencies": {
     "test-fixture": "polymerelements/test-fixture#^0.8.0",
     "iron-iconset": "polymerelements/iron-iconset#^0.8.0",
-    "core-icons": "polymerelements/iron-icons#^0.8.0",
+    "iron-icons": "polymerelements/iron-icons#^0.8.0",
     "iron-doc-viewer": "polymerelements/iron-doc-viewer#^0.8.0",
     "webcomponentsjs": "webcomponents/webcomponentsjs#^0.6.0",
-    "web-component-tester": "~2.2.6"
+    "web-component-tester": "*"
   }
```

Comment:
@nevir can clarify if there is a better choice, but last time he told me that both `^` and `*` are acceptable, just not `~`.